src/utils/aria.test.ts

```diff
@@ -0,0 +1,33 @@
+import { getAriaDescribedByProp } from './aria';
+
+describe('getAriaDescribedByProp', () => {
+  it('is empty when empty array was given', () => {
+    const data = getAriaDescribedByProp([]);
+    const expectation = {};
+    expect(expectation).toEqual(data);
+  });
+
+  it('has the given id', () => {
+    const data = getAriaDescribedByProp(['other-id']);
+    const expectation = { 'aria-describedby': 'other-id' };
+    expect(expectation).toEqual(data);
+  });
+
+  it('has all the given ids', () => {
```

Comment:
Good to have tests for this as well even though the util is quite simple. I'd add a fourth undefined id here in the middle of the array to ensure no additional whitespace is entered / everything is working as expected.

In addition, I think we should start considering dividing the tests into sets. Like CI = all, Hooks = smoke

Reply:
Added the suggested `undefined` between the existing items to this test.

Should I rename this test to `aria.unit.test.ts`?
It can be then run as `yarn test unit`. _Or just like before, `yarn test`_

Reply:
I wouldn't do the dividing / modularization now and the need for it comes more from reducing test running time for commonly repeating tasks like push rather than anything else.

Reply:
Okay, let's leave it for now. 

It really would be nice to have the tests run by git-hooks to be faster.